### PR TITLE
[Feature] Support the team management

### LINKED ISSUE #1729: [Feature] Support Team Management
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/streampark/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

Support Team Management, this is the first subtask of #1629 .

This issue includes team management, team member management, all old projects and apps will be migrated to the default team.

### Usage Scenario

_No response_

### Related issues

#1629

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by 1996fanrui**
## What problem does this PR solve?

Issue Number: close #1729 

Problem Summary:

## What is changed and how it works?

Support the team management, it includes team management, team member management, all old projects and apps will be migrated to the default team.

Feel free to ping committers for the review!

## Contribution Checklist

  - Make sure that the pull request corresponds to a [GITHUB issue](https://github.com/apache/streampark/issues).

  - Name the pull request in the form "[Feature] [component] Title of the pull request", where *Feature* can be replaced by `Hotfix`, `Bug`, etc.

  - Minor fixes should be named following this pattern: `[hotfix] [docs] Fix typo in README.md doc`.

## Purpose of this pull request

Support the team management, it includes team management, team member management, all old projects and apps will be migrated to the default team.

### CODE REVIEW

**Comment by 1996fanrui** [comment]
Hi @wolfboys , this PR is ready, please help review in your free time, thanks a lot~

**Comment by wolfboys** [comment]
> Hi @wolfboys , this PR is ready, please help review in your free time, thanks a lot~

👌I will be review it later.

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/TeamMemberServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+    @Override
+    @Transactional
+    public void deleteUserRolesByUserId(String[] userIds) {
+        Arrays.stream(userIds).forEach(id -> baseMapper.deleteByUserId(Long.valueOf(id)));
+    }
+
+    @Override
+    public IPage<TeamMember> findUsers(TeamMember teamMember, RestRequest request) {
+        AssertUtils.isTrue(teamMember.getTeamId() != null, "The team id is required.");
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/TeamMemberServiceImpl.java:74]: If admin login for the first time and no team selected (no team selected in the front-end), now visit the team-member UI will cause exception. However, this issue can be improved in the next pr.

**Review by wolfboys — APPROVED** [review]
LGTM, There are still some problems on the front-end to be fixed, which can be further followed up in the next pr

**Pull request merged**